test/test_integration_list.py

```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
+
+"""
+CLI List Integration tests
+
+This test file handles the main test flows for the list command
+"""
+import io
+from contextlib import redirect_stderr
+from unittest.mock import patch
+
+import Utils
+from .util.cli_helpers import integration_test
+from test.util.test_runners.integration_test_case import IntegrationTestCaseBase
+
+
+# Not coverage tested since integration tests doesn't run in
+# the same build step
+class CliIntegrationListTest(IntegrationTestCaseBase):  # pragma: no cover
+    def __init__(self, testname, client_params=None, environment_params=None):
+        super(CliIntegrationListTest, self).__init__(testname, client_params, environment_params)
+
+    # *************** HELPERS ***************
+
+    def setup_cli_params(self, env_vars, *params):
+        self.cli_auth_params = ['--debug']
+        self.cli_auth_params += params
+
+        return self.cli_auth_params
+
+    def setUp(self):
```

Comment:
why its called setUP it can't be setup?

Reply:
python convention https://docs.python.org/3/library/unittest.html
this setup is run before each test definition